HW-0.Rmd

```diff
@@ -9,6 +9,6 @@ Your tasks!
 
 * Change the name of the author above to yours
 * Change the date
-* Replace the NHL URL with the URL for the document you just posted to Rpubs **here**: [My RPubs Document](https://www.nhl.com/)
+* Replace the NHL URL with the URL for the document you just posted to Rpubs **here**: [My RPubs Document](http://rpubs.com/DelaneyMoran/154869)
```

Comment:
Good job. Rpubs is a quick way to post R code/output to the web so you can share your work with collaborators.